docs/source/faq.rst

```diff
@@ -296,9 +302,9 @@ SEE ALSO: the :doc:`/using/envs` section of our documentation site.
 
 **Linux, OS X:** ``source deactivate myenv``
 
-**Windows:** deactivate myenv``
+**Windows:** ``deactivate myenv``
 
-NOTE: It is good practice to deactivate one environment before activating another.
+NOTE: In Windows it is good practice to deactivate one environment before activating another.
```

Comment:
Yeah, on windows 'activate a' and then 'activate b' would leave your prompt saying 'b a C:\Users\username>' instead of 'b C:\Users\username>'.